### ISSUE TITLE
func create CLI does not work without language option

### ISSUE THREAD

**Issue by aslom** [knative/func#1508]
From help 

```
func create -h 
...
EXAMPLES
	o Create a Node.js function (the default language runtime) in the current
	  directory (the default path) which handles http events (the default
	  template).
	  $ func create

	o Create a Node.js function in the directory 'myfunc'.
	  $ func create myfunc
```

Examples did not work:

```
func version
v1.8.2

mkdir myfunc
cd myfunc

func create
Error: Required flag "language" not set.
Available language runtimes are:
  go
  node
  python
  quarkus
  rust
  springboot
  typescript

func create -l node
Created node function in /Users/aslom/Documents/awsm/go/src/github.com/aslom/myfunc
```

**Comment by lance** [comment]
@aslom thanks for reporting this. At the moment, I think this is really more of an issue with the help text than it is with the CLI. We have determined in the past that there should be no default for the language when starting a new project for the first time, so the behavior is correct. I believe also that @lkingland is working on some changes to the CLI that will persist the chosen language after the first run of `func create` and defaulting to that value on subsequent runs.

**Comment by aslom** [comment]
SGTM - for conistency it would be good to update the help text to be aligned with the changes?

**Comment by aslom** [comment]
@lkingland @lance as simple as changing https://github.com/knative/func/blob/main/docs/reference/func_create.md ?

**Comment by lance** [comment]
> @lkingland @lance as simple as changing https://github.com/knative/func/blob/main/docs/reference/func_create.md ?

Yes, specifically these two examples. https://github.com/knative/func/blob/132d0492ad75811cb8028d2f81eda85876694467/cmd/create.go#L63-L70
